hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientResult.java

```diff
@@ -183,6 +185,16 @@ public void close() {
         }
     }
 
+    @Override
+    public Object deserialize(Object value) {
+        return service.deserializeRowValue(value);
+    }
+
+    @Override
+    public Object deserialize(LazyTarget value) {
```

Comment:
Can an object of type `LazyTarget` be received by the client? I am asking this because we didn't have such logic on the non-Java clients

Reply:
Doing that is not possible with public API.